security/audit.rst

```diff
@@ -33,6 +32,25 @@ This command also supports using your own JFrog Platform as a private provider f
 vulnerability scanning. See the :ref:`Adding private providers
 <devops_audit_private_providers>` section for more details.
 
+Filtering queried packages
+--------------------------
+
+By default, the ``conan audit scan`` command will query all packages in the dependency graph.
+You can filter the packages to be queried based on their context using the ``--context`` option,
+which accepts ``"host"``, or ``"build"`` as values, and when omitted, defaults to quering both contexts.
+
+This allows you to skip checking for CVEs in build requirements, which are not part of the final product
+and therefore less relevant (but still important!) for vulnerability scanning.
+
+It's also possible to perform this filter using the ``conan audit list`` command,
+by levaring the packages list filtering from the ``conan list`` command. For example:
+
+* Use ``conan graph info`` making sure to also add ``--format=json`` and store the resulting json to a file (``graph.json`` for this example)
+* Run ``conan list --graph=graph.json --graph-context=host --format=json > pkglist.json``
+  - This creates a packages list for the resolved dependency graph, but filters it to only contain the ``host`` context packages using the ``--graph-context`` argument.
+* Now pass the generated ``pkglist.json`` to ``conan audit list --list=pkglist.json``.
```

Comment:
How about using a code block with comments to explain the commands, instead of a written explanation?

Reply:
Looks much nicer now, good idea